share/doc/homebrew/Formula-Cookbook.md

```diff
@@ -303,14 +303,15 @@ correct. Add an explicit [`version`](http://www.rubydoc.info/github/Homebrew/bre
 
 ## Commit
 
-Everything is built on Git, so contribution is easy:
-
+Everything is built on Git, so contribution is easy:  
+1. Fork the Homebrew/homebrew-core repository on GitHub. This creates a pushable, personal remote repository. This is needed as only Homebrew maintainers have push access to the main repository.  
+2. Add the pushable forked repository with `git remote add YOUR_USERNAME https://github.com/YOUR_USERNAME/homebrew-core.git`
```

Comment:
:+1: on the `YOUR_GITHUB_USERNAME` (or `YOUR_GITHUB_HANDLE` if that widely understood) suggestion for greater clarity, also in other places in this document where the GitHub user name is referenced.

I'm not sure about the “all-caps placeholder with surrounding angle brackets” style. My impression when reading documentation was that we (and others) typically do one of the following two, but not both:

- All caps: `YOUR_GITHUB_HANDLE`.
- Lowercase or natural capitalization with angle brackets: `<your-github-handle>` or `<your-GitHub-handle>`.

I'm not particularly tied to any of those styles, but it feels weird combining them. In any case it would be nice to be more consistent with our use of placeholders and I guess we should be picking what is more frequently used in the Homebrew documentation.